src/test/scala/pl/allegro/tech/marathon/plugin/auth/FakeHttpResponse.scala

```diff
@@ -0,0 +1,22 @@
+package pl.allegro.tech.marathon.plugin.auth
+
+import mesosphere.marathon.plugin.http.HttpResponse
+
+class FakeHttpResponse(
```

Comment:
It's just DTO, no logic and purpose is to check how class use Response object. I could use some stub/mock lib but I think this object is simpler.